Denise Escobar Front End Project-Lambda Notes

DenEscobar opened the pull request:
@katiagilligan888

Trello Board:

https://trello.com/b/cER0snho/lambda-notes-denise-escobar

DenEscobar committed:
Starting my project

-I created my trello board and added all the MVP and Extra Features as cards on their respective lists
-I used create react-app to begin my project and deleted extra code

DenEscobar committed:
Added components and basic react set up to each

DenEscobar committed:
Notes are mapping out and I added a css reset

DenEscobar committed:
Added basic styling and classNames

DenEscobar committed:
I am able to create new notes

DenEscobar committed:
Added a deletehandler that is passed down through props to each note and am able to delete notes

DenEscobar committed:
Working on routing my app. I can currently route to the home notes component and also to a seperate create note component

DenEscobar committed:
I can route to a single note

DenEscobar committed:
I can update notes, working more on my styling

DenEscobar committed:
Added more styling to my application

katiagilligan888 commented on lambda-notes/src/components/CreateForm.js:
```diff
@@ -0,0 +1,52 @@
+        e.preventDefault()
+        axios
+            .post("https://fe-notes.herokuapp.com/note/create", {
+                title: this.state.title,
```
Try to use destructuring more often

katiagilligan888 reviewed (commented):
Denise, good job with the project. The code looks good and all the requests function correctly. You still need to add the delete modal as specified in the design files. Also, the Edit Note needs to have the note data in the input fields, instead of blank input fields. If you have time, add responsive styling as well.

katiagilligan888 commented:
Also for the Edit and Create Note views try to add Links on the buttons that route back to the Notes Page for a better user experience.

DenEscobar committed:
Updated front end project to run on new database...
-Changed all he server endpoints to the localhost
-updated id syntax
-CHecked all enpoints for functionality
